HLTrigger/Muon/python/customizeHLTforDisplacedGlobal.py

```diff
@@ -0,0 +1,44 @@
+import FWCore.ParameterSet.Config as cms
+
+def customizeHLTforDisplacedGlobal(process):
+	'''
+	Temporal solution to adapt the HLT global displaced path with a new filter 
+	that selects dSA and dGbl (this customizer will not be neccesary when the 
+	filter is merged in the central repository).
+	'''
+	process.hltL3orL2fL1f0Filtered24Displaced = cms.EDFilter( "HLTMuonL3orL2PreFilter",
```

Comment:
```suggestion
    '''
    Temporal solution to adapt the HLT global displaced path with a new filter 
    that selects dSA and dGbl (this customizer will not be neccesary when the 
    filter is merged in the central repository).
    '''
    process.hltL3orL2fL1f0Filtered24Displaced = cms.EDFilter( "HLTMuonL3orL2PreFilter",
```
See comment above on indentation.